lib/workers/branch/index.ts

```diff
@@ -320,6 +326,83 @@ export async function processBranch(
     } else {
       logger.debug('No updated lock files in branch');
     }
+
+    if (global.trustLevel === 'high') {
+      logger.info(
+        {
+          tasks: config.postUpgradeTasks,
+          allowedCommands: config.allowedPostUpgradeCommands,
+        },
+        'Checking for post-upgrade tasks'
+      );
+      const commands = config.postUpgradeTasks.commands || [];
+      const fileFilters = config.postUpgradeTasks.fileFilters || [];
+
+      if (
+        is.nonEmptyArray(commands) &&
+        is.nonEmptyArray(config.allowedPostUpgradeCommands)
```

Comment:
This `allowedPostUpgradeCommands` check can be removed as we check for it earlier

Reply:
Changed